### PR TITLE
Added chinese language

### PR DESCRIPTION
**Pull request by SuperYYT**
I translated most of the content in the language file
Part of the content I don't think needs translation

### CODE REVIEW

File: resources/messages/chs.ini [hunk 1]
```diff
@@ -0,0 +1,151 @@
+; chs.ini
```
Comment (SOF3) [inline resources/messages/chs.ini:1]: Why is this chs.ini? The ISO639-2 code for Chinese should be `chi` (bibliographic) or `zho` (terminology).

**Review by SOF3 — CHANGES_REQUESTED** [review]

File: resources/messages/chs.ini (hunk 1 above)
Comment (SuperYYT) [reply resources/messages/chs.ini:1]: chs means simplified Chinese and its traditional counterpart is cht. This is a generic language code. https://help.bing.microsoft.com/#apex/bing/en-us/10004/-1

Comment (SOF3) [reply resources/messages/chs.ini:1]: @SuperYYT it isn't. it's a second-level region code. `chs` alone doesn't make sense without `zh`.

Comment (SOF3) [reply resources/messages/chs.ini:1]: But is `chs` used in the client anyway?

Comment (SuperYYT) [reply resources/messages/chs.ini:1]: yeah >_<

**Pull request merged**